test/Common/Record.hs

```diff
@@ -8,25 +8,34 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE OverloadedRecordDot #-}
```

Comment:
We probably need to gate this behind CPP for the GHC version since we support GHC that doesn't have this feature yet

Reply:
I assume gating the whole file makes sense here?